Dockerfile.j2

```diff
@@ -0,0 +1,48 @@
+FROM debian:jessie
+
+## Clean documentation in FROM image
+RUN find /usr/share/doc -depth \( -type f -o -type l \) ! -name copyright | xargs rm || true
+## Clean doc directories that are empty or only contain empty directories
+RUN while [ -n "$(find /usr/share/doc -depth -type d -empty -print -exec rmdir {} +)" ]; do :; done
+RUN rm -rf                              \
+        /usr/share/man/*                \
+        /usr/share/groff/*              \
+        /usr/share/info/*               \
+        /usr/share/lintian/*            \
+        /usr/share/linda/*              \
+        /var/cache/man/*                \
+        /usr/share/locale/*
+
+## Set the apt source
+COPY sources.list /etc/apt/sources.list
+COPY dpkg_01_drop /etc/dpkg/dpkg.cfg.d/01_drop
+RUN apt-get clean && apt-get update
+
+## Pre-install the fundamental packages
+RUN apt-get -y install                  \
+    rsyslog                             \
+    vim-tiny                            \
+    perl                                \
+    python
+
+COPY rsyslog.conf /etc/rsyslog.conf
+
+RUN apt-get -y purge                    \
+    exim4                               \
+    exim4-base                          \
+    exim4-config                        \
+    exim4-daemon-light
+
+## Install common debug-packages
+{% if docker_base_dbgs %}
+RUN apt-get -y install \
+{% for dbg_pkg in docker_base_dbgs.split(' ') -%}
```

Comment:
No need to keep list of these packages in buildimage since they are not built bu it but installed directly from apt instead
Just put a list of packages here and use debug config directly in conditional